.github/workflows/build.yml

```diff
@@ -62,7 +62,7 @@ jobs:
         with:
           extensions: ${{ matrix.operating-system == 'windows-latest' && contains('["8.2","8.3"]', matrix.php) && 'sockets, curl, zip, ffi' || 'sockets, curl, zip, ffi, grpc' }}
           php-version: ${{ matrix.php }}
-          coverage: none
+          coverage: pcov
```

Comment:
I Have a feeling, that this is if x-debug is unavailable that phpunit falls back to requiring pcov; I can't remember the specifics about why.

Reply:
xdebug/pcov is required by phpunit, otherwise we got this error:

> Script phpunit --debug handling the test event returned with error code 1

pcov is faster so I choose it